articles/quickstart/webapp/java-ee/_includes/_setup.md

```diff
@@ -16,7 +16,7 @@ If you are using Maven, add these dependencies to your `pom.xml`:
 <dependency>
     <groupId>com.auth0</groupId>
     <artifactId>mvc-auth-commons</artifactId>
-    <version>(1.0,2.0)</version>
+    <version>[1.0, 2.0)</version>
```

Comment:
These changes (and similar ones below) are to show specifying Maven dependency versions such that it will use latest of version 1.